.github/workflows/format.yml

```diff
@@ -0,0 +1,26 @@
+name: Scalafmt
+
+permissions: read-all
+
+on:
+  pull_request:
+    branches: ['**']
+
+jobs:
+  build:
+    name: Code is formatted
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
```

Comment:
```suggestion
    runs-on: ubuntu-latest
```

Reply:
Thanks for the simplification, I will commit this when just before we actually merge the PR.

Reply:
I have applied this change, I didn't commit the suggestion due to the fact that I am rebasing/modifying the PR as formatting changes update. @spangaer Let me know if you want co authored git attributes or if you don't care because its a pretty trivial commit.

Reply:
Don't sweat it, fine without.

Reply:
It turns out the matrix is needed anyways (the step isn't running for some reasons).

Reply:
That sounds weird

Reply:
So I just put it back in, force pushed the PR and the scalafmt step now appears? I will leave it as it is, its not a big deal anyways.

Reply:
Any chance that you left some line behind?

Reply:
You did
`runs-on: ${{ matrix.os }}`

it still said that, while it had to be

`runs-on: ubuntu-latest`

Reply:
https://github.com/mdedetrich/akka-apache/compare/a31ad604584275c06749c2569aec00dcd800231b..64b41c5689df1eb034719b64bf601f6605e786f5

Reply:
Yes I just saw it now, its working all good.